data/team.yml

```diff
@@ -2146,3 +2146,14 @@ silky_desailly:
   location: Sydney, Australia 🇦🇺
   links: '[LinkedIn](https://www.linkedin.com/in/silkysachdeva)'
   description: 'I am a spirited adventurer, fueled by a passion for travel, a thirst for challenges, and a firm belief that change is the only constant in life.'
+  
+   sara_dam:
```

Comment:
I think indentation here matters? Make sure your Name `sara_dam` is top level, then the rest is indented one level after your name.